src/lib/components/ui/Button/README.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Button
+menu: Components
+---
+
+import { Playground, Props } from 'docz'
+import { Button } from './Button'
+
+# Button
+
+Tohle je popis Buttonu a je to fakt super!
+
+<Props of={Button} />
+
+## Basic usage
+
+<Playground>
```

Comment:
Could we please have working examples? 
![Screenshot from 2020-06-02 09-05-15](https://user-images.githubusercontent.com/4589176/83490509-6c83d580-a4b0-11ea-9c8e-9feb3823a9a4.png)